pkg/aggregator/aggregator.go

```diff
@@ -32,11 +32,14 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/status/health"
 )
 
-// DefaultFlushInterval aggregator default flush interval
-const DefaultFlushInterval = 15 * time.Second // flush interval
-const bucketSize = 10                         // fixed for now
-// MetricSamplePoolBatchSize is the batch size of the metric sample pool.
-const MetricSamplePoolBatchSize = 32
+const (
+	// DefaultFlushInterval aggregator default flush interval
+	DefaultFlushInterval = 15 * time.Second // flush interval
+	// MetricSamplePoolBatchSize is the batch size of the metric sample pool.
+	MetricSamplePoolBatchSize = 32
+
+	bucketSize = 10 // fixed for now
+)
```

Comment:
The change is due to auto `gofumpt` which I use because there is (was?) a Golang linter in `golangci` that was stricter than `gofmt` in some cases, so I'd rather keep the cleaner changes.

Reply:
If there is a code formatting lint that doesn't accept code that is already in the repo, it should be addressed in a separate PR with a clear description. We do have to do code archaeology at times, unfortunately, so having clearly labeled changes is important